addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15811,7 +15811,10 @@ msgctxt "#24162"
 msgid "Use Kodi virtual keyboard"
 msgstr ""
 
-#empty string with id 24163
+#: ./system/settings/settings.xml
+msgctxt "#24163"
+msgid "Remove orphaned dependencies recursively"
```

Comment:
Is this correct English or should the read "Recursively remove ..." ? And is the term "recursively" something a normal user understands or is it by far to technical for John Doe? I would phrase this "Remove all orphaned dependencies"

Reply:
yes we can call it `Remove all orphaned dependencies`.
point is after my test - install/uninstall aeon.silvo - you see only **1** orphaned at `Settings -> System -> Add-ons - Manage dependencies`.... only `script.skinshortcuts`,... and after that one is removed the next 2 orphaned are revealed. but you're right, this is not important to John

Reply:
done